docs/docs/how-to/use-events-on-chart.md

```diff
@@ -148,7 +138,8 @@ import createPlotlyComponent from 'https://cdn.skypack.dev/react-plotly.js/facto
 ```
 
 
-Then, we created a function component called `MyCustomComponent`. This component will render the chart. We use the `createPlotlyComponent` function to create a Plotly component. Then, we create a function called `barOnClick` that will be called when the user clicks on the bar. This function will display an alert message. 
+- Create a function component called `MyCustomComponent`. This component will render the chart. The `createPlotlyComponent` function is used to create a Plotly component. 
+- Then, a function called `barOnClick` is created that will be called when the user clicks on the bar. This function will display an alert message. 
```

Comment:
@2001asjad Can we change this from `Then, a function called barOnClick` to `A function called barOnClick`?